cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -113,6 +112,13 @@
 <!--[if gt IE 9]><!--><link rel="stylesheet" href="{{ static('css/main.css') }}"><!--<![endif]-->
 {% endblock css %}
 
+{#
+    ======
+    PRELOADING DYNAMIC ASSETS
+    ======
+#}
+<link rel="preload" src="{{ static('js/routes/common.js') }}" as="script">
```

Comment:
`src` should be `href` here

Reply:
fixed in https://github.com/cfpb/cfgov-refresh/pull/3713/commits/0dfe88e20bbdbd2b0b6a344db49614419d1d936b